specs/mocks/geolevels.json

```diff
@@ -13,29 +13,29 @@
 }, {
     "id": "fr/canton",
     "name": "French canton",
-    "parents": ["fr/county"]
+    "parents": ["fr/departement"]
 }, {
-    "id": "fr/county",
+    "id": "fr/departement",
     "name": "French county",
     "parents": ["fr/region"]
 }, {
     "id": "fr/district",
```

Comment:
Good question, I didn't made that change because we don't expose these territories (yet?) but maybe we should do that everywhere on the geozones side?